### ISSUE TITLE
i18n

### ISSUE THREAD

**Issue by ghost** [CharlesPockert/aurelia-bs-grid#53]
Is there any support planned for i18n in the near future. It would be a must for me to use it in my application, and i don't really like the idea of changing the strings every time i update to the next version :smile: )
Something like https://github.com/zewa666/aurelia-i18next maybe?

I could help with German translations, or maybe try to implement it using the mentioned module and creating a pull request

**Comment by charlespockert** [comment]
Yeah I'd appreciate it - there's a lot of work to do on this repo and I've just not had the time lately to contribute much. 

Are you looking to translate the static texts for the grid (such as Page 1 of 2 etc) or are you looking at adding some sort of built-in support on grid data/columns etc?

**Comment by ghost** [comment]
No i don't need to translate the grid data, just the static texts. I'll create a pull request in the next days when i find time.
